src/cycax/cycad/cycad_side.py

```diff
@@ -606,9 +846,58 @@ def cylinder(
             diameter: The diameter of the hole.
             height: How tall to make the cylinder.
         """
-        super().cylinder(pos=pos, diameter=diameter, height=height, sink=sink)
+        super().cylinder(pos=pos, diameter=diameter, height=height, sink=sink, original_surface=original_surface)
         self._parent.z_min = self._parent.z_min - height + sink
 
+    def sphere_add(
```

Comment:
Very repetitive code. Creating the methods as features on the sides manually like this is to much work, I'd prefer smarter code.